sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLBoxDAO.java

```diff
@@ -400,13 +400,13 @@ public void emptySingleTube(Box box, String position) throws IOException {
       }
```

Comment:
I'd like to see methods `emptySingleTube` and `emptyAllTubes` renamed to their `discard` counterparts